hazelcast/src/main/java/com/hazelcast/map/impl/tx/TransactionalMapProxySupport.java

```diff
@@ -127,6 +128,23 @@ boolean containsKeyInternal(Data dataKey, Object objectKey) {
         }
     }
 
+    boolean containsValueInternal(Object value, Set<Data> removedKeys) {
+        // TODO: what to do if near cache is enabled?
```

Comment:
You can remove this, seems like even the map implementation doesn't do any optimisations for `containsValue` in the case near cache is enabled.